### PR TITLE
Translation note for Home/end to avoid mistranslations.

### LINKED ISSUE #35608: Translation note requested
https://plugins.trac.wordpress.org/browser/gutenberg/trunk/build/components/index.js#L52427

The string "Home and End" ... Is that refering to a homepage and it's ending OR the actual buttons on the keyboard "Home" and "End"?

I think adding a translation note here or having the "keyboard key" label would help translators.

### PR DESCRIPTION
**Pull request by NekoJonez**
## Description
Should fix #35608 , this is my first time I did a translation note ... So feedback is welcome. 

## Types of changes
Line 134 added.

### CODE REVIEW

File: packages/components/src/date-time/index.js [hunk 1]
```diff
@@ -131,6 +131,7 @@ function DateTimePicker(
 								<abbr aria-label={ __( 'Home and End' ) }>
 									{ __( 'Home/End' ) }
 								</abbr>
+								/* Translators: Home/End reffer to the 'Home' and 'End' buttons on the keyboard.*/
```
Comment (Mamaduka) [inline packages/components/src/date-time/index.js]:
```suggestion
								<abbr aria-label={ __( 'Home and End' ) }>
									{ /* Translators: Home/End reffer to the 'Home' and 'End' buttons on the keyboard.*/ }
									{ __( 'Home/End' ) }
								</abbr>
```

Comment (NekoJonez) [reply packages/components/src/date-time/index.js]: @Mamaduka Implemented. :)

**Comment by Mamaduka** [comment]
Hi, @NekoJonez

Try running `npm run format-js` after the changes, it should fix all the linting issues.

**Comment by NekoJonez** [comment]
> Hi, @NekoJonez
> 
> Try running `npm run format-js` after the changes, it should fix all the linting issues.

I only work in the browser... So, running commands isn't possible for me :/

**Review by Mamaduka — APPROVED** [review]
No worries, I fixed those on my end 👍 

I will try to merge this once all checks are green. Thanks for contributing, and congratulations on your first PR 🎉

**Comment by NekoJonez** [comment]
> No worries, I fixed those on my end 👍
> 
> I will try to merge this once all checks are green. Thanks for contributing, and congratulations on your first PR 🎉

Excellent! Thanks my man! 
And happy to help. 

Btw, is this translation note also going to WP core? Since, the strings are already in core and it would be a shame if we avoid mistranslations in Gutenberg and not in Core for the same strings and features.

**Comment by Mamaduka** [comment]
> Btw, is this translation note also going to WP core?

Yes, this change will be part of 5.9.

**Comment by NekoJonez** [comment]
> > Btw, is this translation note also going to WP core?
> 
> Yes, this change will be part of 5.9.

Lovely! A polygot job well done me thinks!

**Pull request merged**